rand_distr/src/pert.rs

```diff
@@ -47,6 +48,18 @@ pub enum PertError {
     ShapeTooSmall,
 }
 
+impl fmt::Display for PertError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str(match self {
+            PertError::RangeTooSmall => "min..=max is not proper range in PERT distribution",
```

Comment:
Is "proper range" a well-defined thing? Here we require `max > min`; in the Triangular distribution we require `max ≥ min` but you give the same error.

Maybe just say "requirement min < max is not met" ?